solidity/dashboard/src/css/timeline.less

```diff
@@ -0,0 +1,101 @@
+@margin-bottom-between-elements: 1rem;
+@left-border-width: 8px;
+
+.timeline {
+  position: relative;
+
+  &__element {
+    position: relative;
+    min-width: 200px;
+    padding: 1.5rem 2rem;
+    border: 1px solid #AE96E1; // @color-brand-violet-40
+    border-left: @left-border-width solid #633EB1; // @color-brand-violet-80
```

Comment:
Add these brand-violet colors to the `vars.scss` (i added some of them in my PR also):

https://github.com/keep-network/keep-core/blob/e05a336380ac46563ab490d736757b056899fc4d/solidity/dashboard/src/css/vars.less#L10

Reply:
I've added the brand-violet colors in my previous PRs please see 27e9b716b138146ac4b81615b2f5d970d9cad48c. I need to merge `cov-pools-dapp` branch and I'll update here.

Reply:
Kk, i forgot about it. Resolving this convo then.